lib/window-settings.js

```diff
@@ -21,14 +18,14 @@ show_selectedFolders();
  */
 function changeFolders_event() {
```

Comment:
Still underscores. I have a lot of changes to pushed which include a linter file that automatically detects these issues.